src/app/feed/feed-post/feed-post.component.ts

```diff
@@ -95,6 +95,7 @@ export class FeedPostComponent implements OnInit {
   @Input() nftCollectionHighBid = 0;
   @Input() nftCollectionLowBid = 0;
   @Input() isForSaleOnly: boolean = false;
+  @Input() nftLastAcceptedBidAmountNanos: number;
```

Comment:
this doesn't look like an input to the component, but rather a simple attribute since the parent component won't be passing this value. I think we can convert this line to the below
```suggestion
  nftLastAcceptedBidAmountNanos: number;
```